frontend/public/components/utils/dropdown.jsx

```diff
@@ -598,6 +532,43 @@ export class Dropdown extends DropdownMixin {
   }
 }
 
+export const Dropdown = (props) => {
+  const { storageKey } = props;
+
+  // Should be undefined so that we don't save undefined-bookmarks.
+  const bookmarkStorageKey = storageKey ? `${storageKey}-bookmarks` : undefined;
+
+  const [favoriteKey, setFavoriteKey] = useUserSettingsCompatibility(
+    storageKey,
+    storageKey,
+    null,
```

Comment:
Prefer `undefined` because this is what's being saved in the user settings:
```
  dropdown-storage-namespaces: 'null'
```

Reply:
Removed the default so its now undefined.